osbenchmark/results_publisher.py

```diff
@@ -200,14 +210,26 @@ def _publish_service_time(self, values, task):
     def _publish_processing_time(self, values, task):
         return self._publish_percentiles("processing time", task, values["processing_time"])
 
-    def _publish_percentiles(self, name, task, value):
+    def _publish_recall(self, values, task):
+        recall_k = values["recall@k"]
+        recall_1 = values["recall@1"]
+
+        try:
+            return self._join(
+                self._line("Mean recall@k", task, recall_k["mean"], "", lambda v: "%.2f" % v),
+                self._line("Mean recall@1", task, recall_1["mean"], "", lambda v: "%.2f" % v)
+            )
+        except KeyError:
+            return None
```

Comment:
why do we need to catch this? can you return from like 214 if keys doesn't exist?